src/components/messages/notification-preferences.tsx

```
@@ -0,0 +1,658 @@
+            {getChannelIcon(channel)}
+          </div>
+          <span className={`text-sm font-medium ${selectedChannels.includes(channel) ? 'text-blue-900' : 'text-gray-700'}`}>
+            {getChannelLabel(channel)}
+          </span>
+        </label>
+      ))}
+    </div>
+  );
+};
+
+const NotificationPreferences: React.FC<NotificationPreferencesProps> = ({
```

Biome (2.1.2)

[error] Shouldn't redeclare 'NotificationPreferences'. Consider to delete it or rename it.

'NotificationPreferences' is defined here:


(lint/suspicious/noRedeclare)